src/main/java/thePackmaster/powers/rimworldpack/MoodPower.java

```diff
@@ -0,0 +1,190 @@
+package thePackmaster.powers.rimworldpack;
+
+import com.badlogic.gdx.graphics.Color;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.animations.TalkAction;
+import com.megacrit.cardcrawl.actions.animations.VFXAction;
+import com.megacrit.cardcrawl.actions.common.*;
+import com.megacrit.cardcrawl.actions.utility.SFXAction;
+import com.megacrit.cardcrawl.actions.utility.WaitAction;
+import com.megacrit.cardcrawl.cards.status.Burn;
+import com.megacrit.cardcrawl.cards.status.Dazed;
+import com.megacrit.cardcrawl.cards.status.Slimed;
+import com.megacrit.cardcrawl.cards.status.Wound;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.UIStrings;
+import com.megacrit.cardcrawl.powers.BufferPower;
+import com.megacrit.cardcrawl.powers.DuplicationPower;
+import com.megacrit.cardcrawl.vfx.ShineSparkleEffect;
+import com.megacrit.cardcrawl.vfx.combat.VerticalAuraEffect;
+import com.megacrit.cardcrawl.vfx.combat.WhirlwindEffect;
+import thePackmaster.powers.AbstractPackmasterPower;
+import thePackmaster.util.Wiz;
+
+import java.util.Random;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.SpireAnniversary5Mod.modID;
+
+public class MoodPower extends AbstractPackmasterPower {
+    public static final String POWER_ID = makeID(MoodPower.class.getSimpleName());
+    public static final String NAME = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).NAME;
+    public static final String[] DESCRIPTIONS = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).DESCRIPTIONS;
+    public static UIStrings inspirationStrings = CardCrawlGame.languagePack.getUIString(modID + ":Inspirations");
+
+    public MoodPower(AbstractCreature owner, int amount) {
+        super(POWER_ID, NAME, PowerType.BUFF, false, owner, amount);
+
+        canGoNegative = true;
+        if(amount < 0)
+            type = PowerType.DEBUFF;
+        else
+            type = PowerType.BUFF;
+
+        checkForBreaksAndInsps();
+    }
+
+    @Override
+    public void onInitialApplication() {
+        checkForBreaksAndInsps();
+    }
+
+    @Override
+    public void stackPower(int stackAmount)
+    {
+        amount += stackAmount;
+
+        checkForBreaksAndInsps();
+    }
+
+    private void checkForBreaksAndInsps(){
+
+        //NOTE: If I do an action to reduce or increase the amount instead of
+        //just doing it directly, I'd get a stack overflow if someone applies
+        //more than 4 mood at once (e.g by stacking copies of Sanguine)
```

Comment:
Or by playing 4 copies of Burning Passion so all your buffs are multiplied by 16, then playing a Relax that normally adds 2 mood -> 32 mood.

Reply:
If you have 12 energy to spend on pure scaling you've basically already won the fight